docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
```

Comment:
Self-hosted TAG should also work with the next v14 version (v14.3.4). Will update when such version is out.

Reply:
It's out now

Reply:
Not yet AFAIK? The latest is 14.3.3, pre self-hosted support. https://github.com/gravitational/teleport/releases/tag/v14.3.3

Reply:
It's cut and you can download it from https://goteleport.com/download/
Roman will create the release in github